EIPS/eip-3475.md

```diff
@@ -20,133 +19,127 @@ This API standard allows for the creation of any number of bonds type in a singl
 
 Current LP token is a simple ERC-20 token, which has not much complicity in data structure. To allow more complex reward and redemption logic to be built, we need a new LP token standard that can manage multiple bonds, stores much more data and gas efficient. ERC-3475 standard interface allows any tokens on solidity compatible block chains to create its own bond. These bonds with the same interface standard can be exchanged in secondary market. And it allows any 3rd party wallet applications or exchanges to read the balance and the redemption conditions of these tokens. ERC-3475 bonds can also be packed into separate packages. Those packages can in their turn be divided and exchanged in a secondary market.
 
-New functions built in ERC-3475 Multiple Callable Bonds Standard, will allow the users to economize their gas fee spend. Trading and burning of ERC-3475 Bonds will also multiply tokens market cap, helping it to recover from recession period. Existing structures, such as AMM exchanges or lending platform can be updated to recognize ERC-3475 Bonds.
+The functions defined in this EIP provide a gas-efficient solution for creating, trading, and burning bonds. Trading and burning of ERC-3475 Bonds will also multiply tokens market cap, helping it to recover from recession period. Existing structures, such as AMM exchanges or lending platform can be updated to recognize ERC-3475 Bonds.
+
+These bonds can also be batch called for issuance and, in turn, be divided and exchanged in a secondary market.The defined function in this  standard do operations on the serialized array  reads/writes  on the array in the contract, rather than storing  in struct mappings, giving significant benefits in storage and query gas costs.
```

Comment:
```suggestion
These bonds can also be batch called for issuance and, in turn, be divided and exchanged in a secondary market. The defined functions in this  standard do operations on the serialized array, rather than storing in struct mappings, giving significant benefits in storage and query gas costs.
```